ddl/ddl_api.go

```diff
@@ -527,10 +527,11 @@ func (d *ddl) DropSchema(ctx sessionctx.Context, schema model.CIStr) (err error)
 		return errors.Trace(infoschema.ErrDatabaseNotExists)
```

Comment:
Do we need to consider `ModifyColumn`,  `TruncateTable`, `ModifySchemaCharsetAndCollate`, `DropTablePartition`, `ExchangeTablePartition`, `SetDefaultValue`, and so on.

Reply:
Some of them don't even have the concept of schema state(like `truncate table`, `modify charset/collate`), and some involve non-trivial changes(like drop partition, modify columns).

Reply:
How about `ActionDropColumns`?

Reply:
DropColumns will be removed when multi-schema change is supported.